app/helpers/batch_edits_helper.rb

```diff
@@ -1,14 +0,0 @@
-# View Helpers for Hydra Batch Edit functionality
-module BatchEditsHelper
```

Comment:
Why did you move this helper to the Hyrax namespace?  Could this move have gone into a separate PR?